File: client/src/app/_models/userParams.ts
import { User } from './user';

export class UserParams {
  instrument?: string;
  minAge = 18;
  maxAge = 210;
  pageNumber = 1;
  pageSize = 8;
  orderBy = 'lastActive';

  constructor(user: User) {
    this.instrument = '';
  }
}

File: API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams : PaginationParams
    {
   

        public string CurrentUserName { get; set; }
        public int MinAge { get; set; } = 18;

        public int MaxAge { get; set; } = 150;

        public string OrderBy { get; set; } = "lastActive";

        public string Instrument { get; set; }


    }
}